### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by misikir21**
--------This landing page has the following features-------

            - A hero section
            - Featured speaker section 
            - A section of the main programs available 
            - A footer section with respected partners
  ---------what files are there
              [ ] Html
              [ ] Css
              [ ]Javascript 
              [ ]Linters
              [ ]Gitignore
              [ ] Readme
for more watch this video https://www.loom.com/share/3bab9793547b45f4a19b55c3ddc5b642

### CODE REVIEW

File: js/speakers.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    speaker4.classList.remove('speaker-4');
+    seeLessBtn.setAttribute('class', 'seeMoreBtn');
+  }
+
+  function seeLess() {
+    seeLessBtn.setAttribute('class', 'speaker-3 less');
+    speaker3.classList.add('speaker-3');
+    speaker4.classList.add('speaker-4');
+    seeMoreBtn.setAttribute('class', 'seeMoreBtn');
+  }
```
Comment (christianbiring1) [inline js/speakers.js:82]: - [ ] When transitioning from mobile to desktop kindly make sure the user doesn't see this:

### Your design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @misikira , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# REQUEST CHANGES :recycle: 
almost

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` and `title`

## Required Changes ♻️
- [ ] Kindly make sure you have the loom video link included in your PR or README where you present the feature of the project and the code you are more proud of.
- [ ] Even though it's a GitHub issue right now, always include the live demo link of your project in the README.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+      <main>
+        <section class="intro-section">
+            <div class="container">
+                <a href="#" class="menu-icon"><i class="fa-solid fa-bars"></i></a>
+                <div class="about">
+                    <h2 class="intro-preheading">"International Conference LLC"</h2>
+                    <h1 class="main-heading">
+                      International Conference LLC
+                    </h1>
```
Comment (aamirkhan2478) [inline about.html:59]: [OPTIONAL] Please grab all this text here inside a heading `h1` tag for semantic tag purposes. :+1:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @misikira , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

 ## Required Changes ♻️
- [ ] As mentioned by the previous reviewer :point_up_2: please add the loom video link in your PR summary for now your video is not playing so kindly add a video link which is play correctly :slightly_smiling_face: 
 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @misikira , 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow  correctly. ✔
- Your commit messages are descriptive. ✔


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Congratulations on fixing the with menu button on the about page :clap: . However, I believe you forgot the last two changes requested by the previous reviewer. Please to work on them. :point_down: 

1. First he aked you to update the link to your presentation video because the one on the pull request description with is [this one](https://www.loom.com/share/ef47f96004bd4e789d52b60a54e1e997) is not working. We can't play the video. 

Screenshot from 2023-04-27 16-13-11

2. Second he requested that in your readme file for the live demo link instead of the link to your github profile to put a link that leads to the deplayed vesion of you site.

Screenshot from 2023-04-27 16-12-55

### Please before submitting for another review ensure to implement these changes. :ok_hand: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply misYou submitted a project that had failing linters and no pull request summary. Please work on these two items before submitting for review.sed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-You can find a live demo in [HERE](https://github.com/misikira).
+You can find a live demo in [HERE](https://misikira.github.io/Confference-web-app/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Owusu-Desmond) [inline README.md:103]: - [ ] Well done recording the presentation video as requested by the previous code reviewer:+1: Please remember from the project requirement, your presentation video should be a maximum length of 5 minutes but currently the length of your presentation video is almost 8 minutes so I suggest your record another video showcasing your project. :pray: 
 - Remember that the video 
   - is the maximum length of 5 minutes.
   - contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
   - contains a demo of the project, showing the pages in different screen sizes.
   - contains a highlight about a piece of the code the you finds interesting or is proud of making.

- [ OPTIONAL] Why don't you write a professional project description explaining what your project is about in your readme and include the presentation video in the readme. :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @misikira  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Nice project design
:heavy_check_mark: Responsive project 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @misikira , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**